docker/Dockerfile.scancode

```
@@ -1,39 +1,36 @@
-# Copyright (c) 2019-2020 VMware, Inc. All Rights Reserved.
-# SPDX-License-Identifier: BSD-2-Clause
-
-FROM debian:buster
-
-# Install fuse-overlayfs and Tern dependencies
-RUN apt-get update && \
-    apt-get -y install \
-    attr \
-    findutils \
-    git \
-    gnupg2 \
-    jq \
-    python3 \
-    python3-pip \
-    python3-setuptools \
-    tar \
-    util-linux \
-    wget && \
-    echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list && \
-    wget --no-verbose https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_10/Release.key -O - | apt-key add - && \
-    apt-get update && \
-    apt-get -y install \
-    buildah \
-    fuse-overlayfs && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
-# Adjust storage.conf to enable Fuse storage.
-RUN sed -i -e 's|^#mount_program|mount_program|g' -e '/additionalimage.*/a "/var/lib/shared",' /etc/containers/storage.conf
-
-# Install tern
-RUN pip3 install --upgrade pip && \
-    pip3 install --no-cache-dir \
-    tern \
-    scancode-toolkit[full]
-
-ENTRYPOINT ["tern", "--driver", "fuse"]
-CMD ["-h"]
+# Copyright (c) 2019-2020 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+FROM python:3.9-slim-buster as base
+
+FROM base as builder
+
+RUN mkdir /install
+WORKDIR /install
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install --no-warn-script-location --prefix=/install \
+    tern \
+    scancode-toolkit[full]
+
+FROM base
+
+RUN echo "deb http://deb.debian.org/debian bullseye main" > /etc/apt/sources.list.d/bullseye.list \
+    && echo "Package: *\nPin: release n=bullseye\nPin-Priority: 50" > /etc/apt/preferences.d/bullseye \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+    attr \
+    findutils \
+    fuse-overlayfs/bullseye \
+    git \
+    jq \

```

```suggestion
    jq \
    libgomp1 \   
```
Necessary for scancode..  Without it gives an error:

```
...
OSError: (\'libgomp.so.1: cannot open shared object file: No such file or directory\'
....
```